conf/chi.conf.defaults

```diff
@@ -217,3 +217,7 @@ storage=distributed
 #
 # Amount of time from the current time to expire an entry
 expires_in=5s
+
+[namespace fleetdm]
+storage=distributed
```

Comment:
Is it necessary to be distributed ?

Reply:
it would be better.

we only caches the fleetdm hostid <-> packetfence device mac mapping. 
and 
fleetdm token (if they use a username + password auth when calling fleetdm APIs) - but I don't think this will be a common solution for fleetdm users. I think most of them will use an API token instead.

a cross-node cache will avoid unnecessary fleetdm API calls no matter on which node "security_event_trigger" falls on